src/Adapter/MSTest.CoreAdapter/Properties/AssemblyInfo.cs

```diff
@@ -16,6 +18,7 @@
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
+[assembly: TypesToLoad(typeof(MSTestDiscoverer), typeof(MSTestExecutor))]
```

Comment:
I realize this feedback is coming late @Sanan07 but I didn't realize this attribute was already available :) The name of this attribute is a little confusing. `[TestExtensionTypes]` may be a better name.

Reply:
My bad, I didn't add you as a reviewer. I will consider changing the attribute name. Thank you for your feedback.

Reply:
No worries - glad to know the attribute is now available! :)